_posts/2019-07-06-accessibility.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post_1
+title: 09- Accessibility
+date: 2019-07-06 -0400
+categories: podcast
+---
+<div class="iframe-container">
```

Comment:
There's a lot of code duplication for the iframes. What you should do is set a variable in the header of your post files (like this one), with something like `anchor: "https://anchor.fm/randomly-typed/embed/episodes/9---Accessibility-e4hslu/a-aiced1"` and us it in the the post layout with something like 
```liquid
    <div class="iframe-container">
      <iframe src="{{ page.anchor }}" frameborder="0" scrolling="no"></iframe>
    </div>
```